integration/README.md

```diff
@@ -1,193 +1,460 @@
-# Integration testing
+# Integration Testing
 
-## Local setup
+Clerk offers SDKs with bindings for different frameworks, platforms, and runtimes. This is a broad surface area and we often don't have control over those upstream dependencies. In an effort to ensure that all Clerk integrations work as expected, the goals of this integration test suite are:
 
-### Installation
+- Catch accidental breaking changes
+- Test existing flows using our prebuilt components
+- CJS and ESM interop works
+- SDKs behave the same locally and deployed
+- The flows work using Clerk's development instances
 
-If you haven’t run the integration tests before, make sure to run `npm i` from the root of the `javascript` repo so all Playwright dependencies are installed - Playwright will download the browser binaries needed for the integration tests.
+We're using [Playwright](https://playwright.dev/) for our tests, so make yourself familiar with it by reading its documentation.
 
-### Env variables
+## Prerequisites
 
-The integration suite uses env variables to switch between Clerk instances or use 3rd party services to access emails sent to complete auth flows like signing in with email code or email links. Even though not all env variables are needed to run specific tests, we recommend setting them all at the beginning to avoid confusion later.
+Before you begin writing tests, you should already have:
 
-Note that these environment variables will be available to the tests via `process.env.INTEGRATION_INSTANCE_KEYS` on our CICD.
+- Followed the setup instructions in the [contributing guide](../docs/CONTRIBUTING.md)
+- Access to Clerk's 1Password
+- Access to the **Integration testing** organization on Clerk (it owns the Clerk instances used in the tests)
```

Comment:
🤔 I think this is not mandatory. They could use their own instances, but they have to use the same settings. We could provide how many instances we are using and their settings as a way to avoid the need of having access to Clerk 1Password.

Reply:
Right now the target audience is other Clerk employees since we rarely if ever have contributions from other people that would need an integration test. Thus they have these requirements for now - we can change that once we see people contributing more